### PR TITLE
Removed all scss, css and postcss

### PR DESCRIPTION
**Pull request by WGillian**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,24 +32,18 @@
...
     "firebase": "^4.1.3",
     "glob": "7.1.1",
     "ionicons-pre": "^1.0.0-pre",
     "moment": "^2.18.1",
     "next": "^4.2.1",
-    "node-sass": "^4.4.0",
-    "normalize.css": "5.0.0",
-    "postcss-easy-import": "2.0.0",
-    "postcss-loader": "1.3.3",
     "raw-loader": "^0.5.1",
```
Comment (richardgill) [inline package.json]: might be able to delete `raw-loader` too?

File: pages/index.js [hunk 2]
```diff
@@ -17,7 +16,7 @@ export default class Index extends Component {
     return (
       <Page>
         <PageContainer>
-          <style dangerouslySetInnerHTML={{ __html: stylesheet }} />
+          <style dangerouslySetInnerHTML={{ __html: styled }} />
```
Comment (richardgill) [inline pages/index.js]: what happens if you delete this entire line.

previously it was essentially grabbing the contents of `'styles/index.scss'`

and putting that inside a `<style>`

We no longer have any css, so we don't need to do that.

`styled` is a library for writing javascript object styles using css - so it's unrelated to this. passing styled in the line above doesn't make any sense.

File: package.json [hunk 3]
```diff
@@ -32,24 +32,18 @@
     "babel-plugin-styled-components": "^1.1.5",
```
Comment (richardgill) [inline package.json]: there are some more things to do with css in the scripts section of this file.

see:

lint:sass,
format:sass
lint
lint:without:prettier

File: package.json [hunk 4]
```diff
@@ -38,7 +34,6 @@
     "ionicons-pre": "^1.0.0-pre",
```
Comment (WGillian) [inline package.json]: I'm unsure whether this should be in here

File: package.json [hunk 5]
```diff
@@ -32,24 +28,17 @@
     "babel-plugin-styled-components": "^1.1.5",
     "babel-plugin-transform-define": "^1.3.0",
     "babel-plugin-wrap-in-js": "^1.1.0",
-    "bootstrap-sass": "^3.3.5",
     "check-node-version": "^2.1.0",
     "firebase": "^4.1.3",
     "glob": "7.1.1",
     "ionicons-pre": "^1.0.0-pre",
```
Comment (richardgill) [inline package.json]: don't need ionicons (it was used in the styling stuff).

File: package.json [hunk 6]
```diff
@@ -32,24 +28,17 @@
     "babel-plugin-styled-components": "^1.1.5",
     "babel-plugin-transform-define": "^1.3.0",
     "babel-plugin-wrap-in-js": "^1.1.0",
-    "bootstrap-sass": "^3.3.5",
     "check-node-version": "^2.1.0",
     "firebase": "^4.1.3",
```
Comment (richardgill) [inline package.json]: don't need firebase, if any code is using it we should remove it.

File: package.json [hunk 7]
```diff
@@ -32,24 +28,15 @@
...
     "next": "^4.2.1",
-    "node-sass": "^4.4.0",
-    "normalize.css": "5.0.0",
-    "postcss-easy-import": "2.0.0",
-    "postcss-loader": "1.3.3",
-    "raw-loader": "^0.5.1",
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
     "react-modal": "^2.2.1",
     "react-scrollable-anchor": "^0.5.0",
```
Comment (richardgill) [inline package.json]: we should probably eliminate `react-scrollable-ancho`, `react-slick`, `smooth-scroll`

**Pull request closed without merging**